how-it-works/1_protocol/02_consensus.subpage.md

```diff
@@ -5,14 +5,118 @@ shareImage: /img/how-it-works/consensus.600.jpg
 slug: consensus
 ---
 
-# Internet Computer Consensus
-<!-- https://img.youtube.com/vi/vVLRRYh3JYo/0.jpg -->
-[![Watch youtube video](https://i.ytimg.com/vi/vVLRRYh3JYo/maxresdefault.jpg)](https://www.youtube.com/watch?v=vVLRRYh3JYo)
+# Consensus
+
+Each subnet of the IC is its own blockchain that makes progress concurrently to the other subnets and thus runs its own instance of the IC core protocol stack including consensus.
+Recall that the goal of consensus is to produce blocks agreed by the nodes of the subnet with an ordered set of messages to be executed.
+This is crucial so that the upper two layers of the protocol stack – message routing and execution – receive the same inputs in every round on each node.
+
+The IC's consensus protocol is a novel protocol that achieves a high throughput both in terms of messages and message payload as well as a short time to finality.
+Block finality on the IC is cryptographic and thus can have very short latency.
+
+Consensus proceeds in discrete rounds.
+At a high level, a consensus round has the following three phases:
+* *Block making:* Creating and proposing a block containing a list of messages to be executed. In every round at least one node proposes a block, ideally exactly one node, but, if necessary, multiple nodes.
```

Comment:
Punctuation problem: suggested edit:
In every round, at least one node proposes a block.
Ideally, only one node proposes a block, but sometimes several nodes may propose a block (only one of which will eventually be finalized).